bazel_codegen/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.3.1+2
+
+- Added support for running in Dart 2 mode via `--preview-dart-2`.
```

Comment:
I'd phrase this differently. "Fixed runtime errors with running in Dart 2 mode via `--preview-dart-2`."

Reply:
done